brokerapi/brokers/cloudsql/broker.go

```diff
@@ -533,7 +535,7 @@ func (b *CloudSQLBroker) PollInstance(instanceId string) (bool, error) {
 		return false, err
 	}
 
-	return b.PollOperation(instance, op)
+	return b.PollOperation(*instance, op)
```

Comment:
It's unfortunate `PollOperation` doesn't take a pointer. Looking at the function, it seems read only. Would it be possible to update it?

Reply:
I'm seeing a lot of instances of struct passing of `models.ServiceInstanceDetails` 

Maybe it would be better if we created an issue and handled this in another PR?